File: HIT/src/HIT.REST/Infrastructure/Entities/Candidate.cs
namespace HIT.REST.Infrastructure.Entities;

public class Candidate : BaseEntity
{
    public string FirstName { get; set; }

    public string LastName { get; set; }

    public string EmailAddress { get; set; }

    public string LinkedInURL { get; set; }

    public CandidateType CandidateType { get; set; }

    public string CandidateLocation { get; set; }

    public int YearsOfExperience { get; set; }

    public IEnumerable<CandidateSkill> CandidateSkills { get; set; }
}

public enum CandidateType
{
    APPLIED,
    FUTURE,
    SEARCHED
}

File: HIT/src/HIT.WEB/src/app/shared/models/candidate.ts
import { CandidateType } from "../enum/CandidateType";

export interface Candidate {
    id?: number;
    firstName: string;
    lastName: string;
    emailAddress: string;
    linkedInURL: string;
    candidateType: CandidateType;
    candidateLocation: string;
    YearsOfExperience: number;
  }
  
